src/js/rx/components/GlossaryPage.jsx

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import BackLink from '../components/BackLink';
```

Comment:
If `BackLink` isn't being used anywhere anymore, let's remove the component too.